src/teleop/gui/src/components/SAArmControls.vue

```diff
@@ -69,6 +75,25 @@ export default {
   methods: {
     updateArmEnabled: function (enabled) {
       this.arm_enabled = enabled;
```

Comment:
Should probably only do this once there is a success right? Or change it back on failure